### ISSUE TITLE
Cannot Open project from VS Code

### ISSUE THREAD

**Issue by rbarna1** [joeferraro/MavensMate#809]
* **MavensMate Desktop Version**:
Beta .6

VSCode 1.6

0.4.2

Win 10 x64

no

37

na35

With MM open on my desktop and with or without my existing project Open, the MM plugin in VSCode always reports on open: "[STATUS]	Instantiating without Project".
The MavensMate:Open Project command fails with the following:
"Running the contributed command:'mavensmate.openProject' failed."

Yet the MavensMate: Open Global Settings command succeeds and opens the settings in the desktop app...so there is *some* connection between VS Code and MM desktop.  Would appreciate some help because obviously I cant save or refresh any files to/from the server without a recognized Open Project.

Note: I think the project itself is configured nicely...it works in the Sublime editor.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/196929402075618/200686542318651)

**Comment by kidtsunami** [comment]
@rbarna1 can you take a screenshot of the explorer pane after you've opened the project in question (with the config sub-folder expanded, example screenshot below)

![_settings_-_sunovion_-_profile_enhancements_-_test_and_cannot_open_project_from_vs_code_ _issue__809_ _joeferraro_mavensmate](https://cloud.githubusercontent.com/assets/504253/19650879/c92c37be-99d8-11e6-9781-84d290da5a2d.png)

And can you include the contents of the .settings file? Feel free to sanitize anything you feel needs to be (I use skitch for screenshots, it's pretty nice).

**Comment by rbarna1** [comment]
I cannot!  This morning I tried wiping out everything (deleting the folder that was the project root that was working for the Sublime->MM connection), and starting fresh, and don't you know, it worked.
So *something* about the existing solution was preventing the VSCode integration from functioning at all.  After starting fresh and creating a new project from within MM desktop there were buttons to open it with either Sublime or VS Code.  I pressed VS Code, and it redirected me to VSCode and all is well.
FWIW, my config sub-folder now looks just like your screenshot, except I don't have org_metadata.

Thanks again
